pkg/metrics/metrics.go

```diff
@@ -92,6 +92,21 @@ var (
 	},
 		[]string{"outcome"})
 
+	// EndpointRegenerationTime is the total time taken to regenerate endpoint
+	EndpointRegenerationTime = prometheus.NewCounter(prometheus.CounterOpts{
+		Namespace: Namespace,
+		Name:      "endpoint_regeneration_seconds",
+		Help:      "Total sum of successful endpoint regeneration times",
+	})
+
+	// EndpointRegenerationTimeSquare is the sum of squares of total time taken
+	// to regenerate endpoint.
+	EndpointRegenerationTimeSquare = prometheus.NewCounter(prometheus.CounterOpts{
+		Namespace: Namespace,
+		Name:      "endpoint_regeneration_square_seconds",
+		Help:      "Total sum of squares of successful endpoint regeneration times",
```

Comment:
Same as above.

Reply:
This change was part of one of the review suggestions... It was originally `time taken to ....` and was asked to be changed to this...